test/torch_np/test_basic.py

```diff
@@ -561,14 +561,19 @@ def test_set_default_float(self, dt):
 @skip(_np.__version__ <= "1.23", reason="from_dlpack is new in NumPy 1.23")
```

Comment:
This test is skipped `@skip(_np.__version__ <= "1.23", reason="from_dlpack is new in NumPy 1.23")` anyway, did you mean to enable it?